components/src/overrides/editform/utils.ts

```diff
@@ -0,0 +1,13 @@
+import EditFormUtils from 'formiojs/components/_classes/component/editForm/utils';
+
+const originalLogicVariablesTable = EditFormUtils.logicVariablesTable;
+
+// decorate this function to include global updates to evalContext.
+function logicVariablesTable(additional = '') {
+  const customEval =
```

Comment:
Looks like this is adding "token" twice to the "Calculated Value" section

![image](https://user-images.githubusercontent.com/17445138/144686577-4a9773e5-5988-4a31-9f5a-d72d9666daaa.png)

Reply:
hmm I actually don't get the token line showing up for me, I wonder why you had the original show up originally.

Reply:
Do you still have a localstorage variable set? Maybe that is still updating this table somehow.

Reply:
Now realised it's a different table, checking to see how to avoid this..